composer.json

```diff
@@ -9,5 +9,11 @@
         "phpunit/phpunit": "^4.8"
     },
     "license": "MIT",
-    "minimum-stability": "stable"
+    "minimum-stability": "stable",
+    "autoload": {
```

Comment:
Can we please make both functions reside in the same file?

Reply:
@Ayesh, sorry for my slow response.

I think it will be the `array_key_first_last.php` file?

Reply:
I'm sorry for the late review :)
`array_key_first_last.php` sounds good to me too. Thank you!

Reply:
I will do that on next PR :).